cli/package.json

```diff
@@ -125,42 +125,42 @@
       "require": "./index.js"
```

Comment:
The weirdness in this file may have been fixed in [this PR](https://github.com/cypress-io/cypress/pull/26743)? Maybe try pulling latest from develop and re-run the `yarn` install